packages/devtools_app/lib/src/provider/provider_nodes.dart

```diff
@@ -49,9 +49,13 @@ final _rawProviderIdsProvider =
       eval.getInstance(idRef, isAlive)
   ]);
 
-  return [
-    for (final idInstance in providerIdInstances) idInstance.valueAsString,
-  ];
+  final ret = <String>[];
+  for (final idInstance in providerIdInstances) {
+    if (idInstance != null) {
+      ret.add(idInstance.valueAsString);
```

Comment:
Hum this is weird that "null" is received here

In which case does this happen?

Reply:
Just using the https://github.com/flutter/samples/tree/master/provider_shopper
sample.